src/channel-manager/index.ts

```diff
@@ -1,7 +1,7 @@
 import LuminoClient from "../lumino-client";
 import {SetupChannel} from "../types/setup";
 import {Node, LuminoNode, NodeList} from "../types/node";
-import { getTokenAddress, toWei } from "../util/token";
+import { tokenAddresses, toWei } from "../util/token";
```

Comment:
We shouldn't use hardcoded token addresses since we can retrieve it from the node directly, we could use that instead.

Reply:
i agreen entirely, this should be covered under [RLP-1084](https://jirainfuy.atlassian.net/browse/RLP-1084):

> algunos parámetros en los test definidos en las topologías o environments estaban siendo extraídos a través de consultas del SDK, lo cual es innecesario. tampoco deben ser hard-codeados; deben ser leídos de las topologías o environments en sí.